src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5279,6 +5281,8 @@ internal static SessionStateAliasEntry[] BuiltInAliases
                     new SessionStateAliasEntry("wjb",
                         "Wait-Job",  "", ScopedItemOptions.AllScope),
 #if !CORECLR
+                    new SessionStateAliasEntry("cfs",
+                        "ConvertFrom-String", "", ScopedItemOptions.AllScope),
```

Comment:
shouldn't be added here. `ConvertFrom-String` uses `AliasAttribute` to declare the alias.
```
PS:3> $t = gcm  ConvertFrom-String | % ImplementingType
PS:4> $t.GetCustomAttributes($false) | ? { $_.GetType().Name -eq "AliasAttribute" }

AliasNames TypeId
---------- ------
{CFS}      System.Management.Automation.AliasAttribute
```